test/stacktrace/test.cpp

```diff
@@ -0,0 +1,62 @@
+#include <algorithm>
```

Comment:
Are you sure this is necessary? There's a `stacktrace_unittest.cc` already. Either way, the naming scheme does not follow glog conventions.